package.json

```diff
@@ -24,9 +24,11 @@
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.26.0",
     "jsdoc": "^3.6.7",
-    "parcel": "^2.0.0-rc.0",
+    "parcel": "^2.8.3",
     "postcss": "^8.3.8",
     "prettier": "^2.4.1",
+    "process": "^0.11.10",
+    "punycode": "^1.4.1",
```

Comment:
Not sure why, but process and punycode get added automatically when running `npm run dev`. The application throws errors without them.